src/logger/logger.ts

```diff
@@ -53,3 +53,16 @@ function getLogger(): Logger {
         ],
     });
 }
+
+export async function logErrorAndGetErrorResponse({
```

Comment:
I'm not sure this belongs in this file - this seems specific to route handlers as it's returning a Response. Can we put this back in `auth/reset-password`?

Reply:
sure - I only moved it out because it was common to both routes, back when I had the invite route. that's gone anyway now